satpy/etc/enhancements/generic.yaml

```diff
@@ -930,3 +930,10 @@ enhancements:
           factor: 21.0
           min_stretch: 0.0
           max_stretch: 20.0
+
+  msg_quicklook:
+   standard_name: msg_quicklook
+   operations:
+     - name: stretch
+       method: !!python/name:satpy.enhancements.stretch
+       kwargs: {stretch: crude}
```

Comment:
For better performance, the min/max values should be defined if at all possible. If they are not defined, `crude` stretch needs to compute the min/max values from the whole data and in effect load it in memory.

There's also a missing new line at the end of the file.

Reply:
done min/max values